src/zzz_od/operation/choose_predefined_team.py

```
@@ -1,124 +1,584 @@
+                    ),
+                )
+                if team_name is None or team_name_mr is None:
+                    log.info(
+                        '预备编队扫描卡位:列:%d 行:%d 未识别队名',
+                        title_x,
+                        title_y,
+                    )
+                    continue
+                if team_name in self.scanned_team_name_set:
+                    log.info(
+                        '预备编队扫描卡位:列:%d 行:%d 队名:%s 已扫描，跳过',
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)